app/services/attribute_asserter.rb

```diff
@@ -44,9 +44,11 @@ def default_attrs
   end
 
   def add_bundle(attrs)
+    use_ascii = bundle.include?(:ascii)
```

Comment:
it seems like this could easily be its own method since `bundle` is available outside of this method?